WooCommerce/Classes/Extensions/UITabBar+Appearance.swift

```diff
@@ -10,5 +10,29 @@ extension UITabBar {
         let appearance = Self.appearance()
         appearance.barTintColor = .appTabBar
         appearance.tintColor = .text
+
+        /// Some iOS 13.x versions don't render the tabbar shadow color correcly while in dark mode.
+        /// To fix it, we have to specifically set it in the `standardAppearance` object.
+        /// Which forces us to set other properties like `badgeBackgroundColor` and `badgeTextAttributes` in the `UITabBarItemAppearance` layouts.
+        ///
+        if #available(iOS 13.0, *) {
+            let standardAppearance = UITabBarAppearance()
+            standardAppearance.backgroundColor = .appTabBar
+            standardAppearance.shadowColor = .systemColor(.separator)
+            applyItemWooAppearance(to: standardAppearance.inlineLayoutAppearance)
+            applyItemWooAppearance(to: standardAppearance.stackedLayoutAppearance)
+            applyItemWooAppearance(to: standardAppearance.compactInlineLayoutAppearance)
+            appearance.standardAppearance = standardAppearance
```

Comment:
I suggest we hide this in a `createTabBarAppearance()` static method so that this block would only contain: 

```suggestion
            appearance.standardAppearance = createTabBarAppearance()
```

This makes it clearer to see what is being done if the OS is > 13.0 while the details are hidden somewhere else.